src/js/index.js

```diff
@@ -663,28 +672,6 @@ export const generate = (baseSpacing = 24, scale = 6) => {
           styles.push('opacity: 0.6;');
```

Comment:
why do we need this style for button disable state. We can set opacity in button > disabled > opacity for disabled state. Please do check this.